devicemap-schema.json

```diff
@@ -84,7 +84,7 @@
           "properties": {
             "name": {
               "type": "string",
-              "pattern": "^lte-[0-9]+$"
+              "pattern": "^lte-[0-9]+(-[0-9]+)?$"
```

Comment:
Is there any concept of slot with lte interfaces? We can keep this for consistency, but wondering if that will cause more confusion @agrawalkaushik @reidstidolph . If slot for lte doesn't make sense, then we should undo that change in the devicemap

Reply:
everything should follow the <type>-<slot>-<port> convention for consistency sake IMO. This could be useful when we have to deal with dual LTE in the future.

Reply:
I assume `lte-x-y` or `lte-x` only impacts the device (KNI) name, the target-interface name will always be `wwp0s21u1i8` for a single LTE system, right?

Reply:
> I assume lte-x-y or lte-x only impacts the device (KNI) name

correct